application/Profile/DBProfile.py

```
@@ -67,13 +73,17 @@
         # key: user_id, value: True (to indicate the entry exists; can be any dummy value)
-        self.resend_cooldown = TTLCache(maxsize=1024, ttl=RESEND_COOLDOWN_TTL_SECOND)
+        self.resend_cooldown = TTLCache(maxsize=MAX_PENDING_ACTIVATION_NUM, ttl=RESEND_COOLDOWN_TTL_SECOND)
 
-        with open('/var/www/ictrl/application/resources/activation_email_template.html', 'r') as f:
-            self.activation_email_body_template = f.read()
+        try:
+            with open(ACTIVATION_EMAIL_TEMPLATE) as f:
+                self.activation_email_body_template = f.read()
+        except OSError as e:
+            logger.exception('Failed to open "%s"', ACTIVATION_EMAIL_TEMPLATE)
+            raise e
```

Ruff (0.8.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)